corehq/messaging/scheduling/models/content.py

```diff
@@ -147,7 +147,7 @@ def send(self, recipient, logged_event, phone_entry=None):
             date=datetime.utcnow(),
             couch_recipient_doc_type=logged_event.recipient_type,
             couch_recipient=logged_event.recipient_id,
-            messaging_subevent_id=logged_event.pk,
+            messaging_subevent_id=logged_subevent.pk,
```

Comment:
Each of the others should be `logged_subevent` too, since in theory the recipient is linked with the subevent, not the parent.

Reply:
Got it, I didn't know that. Thanks! 
https://github.com/dimagi/commcare-hq/compare/kev/sms-email-recipient-id?expand=1